CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## [1.2.46](https://github.com/ably/ably-java/tree/v1.2.46)
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.45...v1.2.46)
+
+**Implemented enhancements:**
+
+- New `action`, `serial`, `createdAt` `Message` fields has been added
```

Comment:
```suggestion
- New `action`, `serial`, `createdAt` `Message` fields have been added.
```

Reply:
I think we also need to add a notice that these fields are new/experimental/not-stable etc.